### ISSUE TITLE
add blur effect

### ISSUE THREAD

**Issue by ky0n** [Guake/guake#2079]
add blur effect as an option 

and feathub seems to be offline.

**Comment by Davidy22** [comment]
I feel like that might be something for the compositor but can look into it.

I haven't checked up on feathub in a bit but it seems like it's been down for a while from what I'm seeing in mentions of it online. I'm ok with just dropping it and sticking to github issues, I didn't quite get why we'd have two bug report pages when I came on board.

**Comment by Rapti** [comment]
If you're using KDE, you can achieve a blur effect like this:
`xprop -f _KDE_NET_WM_BLUR_BEHIND_REGION 32c -set _KDE_NET_WM_BLUR_BEHIND_REGION 0 -id $(xdotool search --name "Guake!" | tail -n1)`
Put it somewhere where it gets executed whenever Guake has started.

I agree that it would be great to have this supported by Guake itself.

**Comment by believer-ufa** [comment]
Yes, transparent background with blurred content would look really nice.

**Comment by danieleftodi** [comment]
In Gnome, add: https://extensions.gnome.org/extension/3193/blur-my-shell/

and tweak it ... and it works:

![Screenshot from 2023-10-31 12-45-29](https://github.com/Guake/guake/assets/35347921/2ede729a-eced-4489-8dd8-5969f6b7e465)
